src/ingest/src/test/scala/osmesa/MultiPolygonRelationReconstructionSpec.scala

```diff
@@ -28,29 +34,29 @@ trait SparkPoweredTables extends Tables {
         .setIfMissing("spark.master", "local[*]")
         .setIfMissing("spark.serializer", classOf[KryoSerializer].getName)
         .setIfMissing("spark.kryo.registrator", classOf[KryoRegistrator].getName)
-    )
-    .getOrCreate()
+    ).getOrCreate()
+  spark.withJTS
 
-  def relation(relation: Int): (Fixture) = Fixture(relation, orc(s"relation-$relation.orc"), wkt(s"relation-$relation.wkt"))
+  def relation(relation: Int): Fixture = Fixture(relation, orc(s"relation-$relation.orc"), readWKT(s"relation-$relation.wkt"))
 
   def orc(filename: String): DataFrame = spark.read.orc(getClass.getResource("/" + filename).getPath)
 
-  def wkt(filename: String): Seq[String] = {
-    try {
-      Source.fromInputStream(getClass.getResourceAsStream("/" + filename)).getLines.toSeq match {
-        case expected if expected.isEmpty => Seq("")
-        case expected => expected
-      }
-    } catch {
-      case _: Exception => Seq("[not provided]")
+  def readWKT(filename: String): Seq[jts.Geometry] = {
+    Source.fromInputStream(getClass.getResourceAsStream("/" + filename)).getLines.toSeq match {
+      case expected if expected.isEmpty =>
+        Seq()
+      case expected =>
+        expected.map(wktReader.read)
     }
+   // } catch {
+   //   case _: Exception => Seq("[not provided]")
```

Comment:
I added this so that this test could also be used to generate expected WKT if no fixture is available; mind re-enabling it?

Reply:
Yeah, one reason I've not wanted to hit *fire* on this is the difference in counts. I'm pretty certain the semantics of these operations hasn't changed, so I think your hunch is right